modules/imaging_browser/php/mrinavigation.class.inc

```diff
@@ -45,12 +45,11 @@ class MRINavigation
     function __construct($sessionID = null)
     {
         $this->_sessionID           = $sessionID;
-        $this->_FilteredSessionList = $_SESSION['State']
-            ->getProperty('mriSessionsListed');
+        $this->_FilteredSessionList
+            = is_array($_SESSION['State']->getProperty('mriSessionsListed'))
+            ? $_SESSION['State']->getProperty('mriSessionsListed')
+            : [];
```

Comment:
I don't think this is a good use of the ternary operator. It makes the code too opaque. An if/else would be easier to read.